modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -232,6 +234,19 @@ def validate_form_526_veteran_homelessness!
       end
     end
 
+    def validate_service_after_13th_birthday!
+      age_thirteen = auth_headers&.dig('va_eauth_birthdate')&.to_datetime&.next_year(13)&.to_time
```

Comment:
Here and below: do we need so many safe navigation operators? Or can we use guards to return early / raise errors if some parameters are missing?